templates/docs/examples/patterns/buttons/icon.html

```diff
@@ -4,7 +4,9 @@
 {% block standalone_css %}patterns_buttons{% endblock %}
 
 {% block content %}
-<button class="p-button--positive has-icon"><i class="p-icon--minus is-light"></i></button>
+<button class="p-button--positive has-icon"><i class="p-icon--success"></i></button>
+<button class="p-button--negative has-icon"><i class="p-icon--success"></i></button>
```

Comment:
This example is supposed to demonstrate different uses of icons in buttons, I wouldn't turn it into success icon exploration.
So leave just the first positive button with the success icon, and drop the rest.

We have separate example https://vanilla-framework-3410.demos.haus/docs/examples/templates/tick-element-comparison where we explore different form elements, so I guess we can put buttons with success icons alongside the buttons there?

<img width="1274" alt="Screenshot 2020-11-17 at 14 28 47" src="https://user-images.githubusercontent.com/83575/99396018-85a70400-28e1-11eb-9e93-f5fd3758e880.png">